### ISSUE TITLE
Have trouble converting this timedtext

### ISSUE THREAD

**Issue by ghost** [ajanicij/yt-timedtext-convert#1]
I got it works well with your example,it said 'finished' etc\
However i downloaded this timedtext from youtube 
https://raw.githubusercontent.com/yogaskung/yt-timedtext-convert/master/timedtext.xml
and did the same
it just kept on converting?

**Comment by ajanicij** [comment]
The format of that file is slightly different from what the script expects. I'll fix the script as soon as I have the time.

**Comment by ghost** [comment]
Thanks for your time looking into this.
But here the [new script](https://github.com/ajanicij/yt-timedtext-convert/commit/ce7e5a763fef360891501931ab6ba99f6f9a399d)  fails to output any data,when I download it firefox says 'application/text (0 bytes)' and that's exactly empty.

**Comment by ajanicij** [comment]
Yes, I know. Sorry for taking so long, but I will fix it. If it's any consolation, I know exactly where the problem is - the format of the timedtext file is different from the one I originally worked with. I will let you know when it's fixed.
